src/presentation/jsend-response-formatter.ts

```diff
@@ -1,26 +1,37 @@
 import { HttpResponseFormatter } from './interfaces/http-response-formatter'
 import { ErrorCode } from '../model/enums/error-code'
+import { Exception } from '../model/exceptions/exception'
+
+enum JSendStatus {
```

Comment:
This seems to be the status of the request so a more apt named could be `RequestStatus`.

Sidenote: What's the difference between `ERROR` and `FAIL` and when should I use one over the other? They seem the same to me. Is error equivalent to status codes 500s and fail is for 300 to 499?

Reply:
Updated to be named `RequestStatus`.

To me from reading the [JSend document](https://github.com/omniti-labs/jsend), it sounds like error is for when an actual exception of some kind is raised. On the other hand fail sounds like it should be used if a call was successful, but some form of validation failed. The example on the document returns a fail if a form field fails validation.

Where to draw the line between error and fail, does indeed seem a bit blurry. Not sure if that means we should remove fail, or not?